subworkflows/local/hic_mapping.nf

```diff
@@ -197,26 +197,32 @@ workflow HIC_MAPPING {
     ch_versions         = ch_versions.mix( GET_PAIRED_CONTACT_BED.out.versions )
 
     //
-    // LOGIC: PREPARE JUICER TOOLS INPUT
-    //
-    GET_PAIRED_CONTACT_BED.out.bed
-        .combine( dot_genome )
-        .multiMap {  meta, paired_contacts, meta_my_genome, my_genome ->
-            paired      :   tuple([ id: meta.id, single_end: true], paired_contacts )
-            genome      :   my_genome
-            id          :   meta.id
-        }
-        .set { ch_juicer_input }
-
-    //
-    // MODULE: GENERATE HIC MAP
-    //
-    JUICER_TOOLS_PRE(
-        ch_juicer_input.paired,
-        ch_juicer_input.genome,
-        ch_juicer_input.id
-    )
-    ch_versions         = ch_versions.mix( JUICER_TOOLS_PRE.out.versions )
+    // LOGIC: SECTION ONLY NEEDED FOR TREEVAL VISUALISATION, NOT RAPID ANALYSIS
+    //
+    if (workflow.commandLine.contains('-entry FULL')) {
```

Comment:
Why not this, since you're setting `params.entry` in other places ?
```suggestion
    if (params.entry == 'FULL') {
```

Reply:
Done.